source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -40,11 +40,30 @@ values from resolv.conf will be used. If either of them is used, it
 will be passed to VDSM overriding configuration from resolv.conf. For
 example we can set DNS configuration in a Network related to certain DC
 (or later to certain Cluster) and this configuration will override
-configuration in resolv.conf. If we now setup configuration in
+configuration in resolv.conf. Note: when Network is being attached in 
+certain host, NetworkAttachment record is created for this, and 
+dns configuration is copied from Network record to NetworkAttachment 
+record. There we can further modify it. If we now setup configuration in
```

Comment:
In QoS, or VLAN we perform a multi-host action to update all attached hosts with the new value. Why not do it so with DNS as well? Even if the reason is the willingness to keep this feature small and simple, please state it in the feature page.